skt/reporter.py

```diff
@@ -234,6 +234,24 @@ def __init__(self, cfg):
         # multireporting
         self.multi_job_ids = []
 
+    def get_config_local(self):
+        """Retrieve a local config file from the disk."""
```

Comment:
I think this needs to say "the local config file", and we need to define better where that file comes from and when it's created.

Reply:
Fixed.